ssv/spectest/tests/runner/postconsensus/too_many_roots.go

```diff
@@ -25,7 +25,7 @@ func TooManyRoots() tests.SpecTest {
 				),
 				Duty: testingutils.TestingAttesterDuty,
 				Messages: []*types.SignedSSVMessage{
-					testingutils.SignPartialSigSSVMessage(ks, testingutils.SSVMsgCommittee(nil, testingutils.PostConsensusAttestationTooManyRootsMsg(ks.Shares[1], 1, testingutils.TestingDutySlot))),
+					testingutils.SignPartialSigSSVMessage(ks, testingutils.SSVMsgCommittee(ks, nil, testingutils.PostConsensusAttestationTooManyRootsMsg(ks.Shares[1], 1, testingutils.TestingDutySlot))),
 				},
 				OutputMessages:         []*types.PartialSignatureMessages{},
 				BeaconBroadcastedRoots: []string{},
```

Comment:
why noe error here?

Reply:
If I'm not mistaken, the committee runner doesn't validate the number of psig msgs received. The only limitation is the message structure itself which allows up to 1000.

Reply:
add TODO here to add validation